internal/datastore/common/sql.go

```diff
@@ -18,6 +18,9 @@ import (
 )
 
 var (
+	// ObjCaveatNameKey is a tracing attribute representing a caveat name
+	ObjCaveatNameKey = attribute.Key("authzed.com/spicedb/sql/objCaveatName")
```

Comment:
Should probably just be `caveatName` since it doesn't apply to the resource or the subject, but rather the relationship as a whole